project/.gitattributes.twig

```diff
@@ -0,0 +1,2 @@
+.* export-ignore
+*.md export-ignore
```

Comment:
@rande Talk about keeping the docs folder, not the md files. Plus, the `README.md` file is just a generic one with a link pointing to the online documentation.

So let's keep as is for now and launch a vote for docs folders.